components/UI/navigation/NavBar.tsx

```diff
@@ -127,7 +127,7 @@ function NavBar() {
             >
               <MenuItem onClick={handleLogout}>Log out</MenuItem>
             </Menu>
-            <ConnectWalletButton />
+            {<ConnectWalletButton />}
```

Comment:
need { } ?

Reply:
ahh, forgot to remove. I tried making it refresh and forgot to remove the brackets